driver/src/contracts/stablex_contract.rs

```diff
@@ -68,11 +75,43 @@ impl StableXContract for StableXContractImpl {
         Ok(parse_auction_data(packed_auction_bytes, index))
     }
 
+    fn get_solution_objective_value(
+        &self,
+        batch_index: U256,
+        orders: Vec<Order>,
+        solution: Solution,
+    ) -> Result<U256> {
+        let (prices, token_ids_for_price) = encode_prices_for_contract(solution.prices);
+        let (owners, order_ids, volumes) =
+            encode_execution_for_contract(orders, solution.executed_buy_amounts);
+
```

Comment:
Do 2 lines of code sharing already justify factoring it into an own function? Probably not, :)

Reply:
I think not in this case. I can create a new function if you feel like it will make the code cleaner though.

Reply:
I think not in this case. I can create a new function if you feel like it will make the code cleaner though.